packages/eas-cli/src/run/android/systemRequirements.ts

```diff
@@ -0,0 +1,45 @@
+import spawnAsync from '@expo/spawn-async';
+import chalk from 'chalk';
+import { existsSync } from 'fs-extra';
+
+import { getAaptExecutableAsync } from './aapt';
+import { adbExecutable } from './adb';
+import { emulatorExecutable } from './emulator';
+
+async function whichAsync(executable: string): Promise<string | null> {
+  const { stdout } = await spawnAsync('which', [executable]);
+
+  if (!stdout || stdout.includes('not found')) {
+    return null;
+  }
+
+  return stdout.trim();
+}
+
+function assertExecutableExists(executable: string): void {
+  if (executable.includes('/')) {
+    if (!existsSync(executable)) {
+      throw new Error(
+        `Couldn't find ${chalk.bold(
+          executable
+        )} executable in the Android SDK. Please make sure it's installed.`
```

Comment:
Is it me or this sounds a little weird?

"Please make sure it's installed." - I'm not sure what is "it" in this sentence.